geode-membership/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitorJUnitTest.java

```diff
@@ -999,6 +1000,52 @@ public void testDoTCPCheckMemberWithUnkownStatus() throws Exception {
     executeTestDoTCPCheck(GMSHealthMonitor.ERROR + 100, false);
   }
 
+  @Test
+  public void heartbeatOversleepCausesWarning() {
+    testHeartbeatSleepScenario(period -> 2 * period + 1,
```

Comment:
maybe replace the "2"s in this test with the logical-interval constant from GMSHealthMonitor

Reply:
actually that's a whole different `2`. This `2` is the (arbitrary) oversleep criterion. Will fix in next commit…